README.md

```diff
@@ -1,16 +1,28 @@
-# OpenTelemetry-Rust
+![OpenTelemetry — An observability framework for cloud-native software.][splash]
+
+[splash]: https://raw.githubusercontent.com/open-telemetry/opentelemetry-rust/master/assets/logo-text.png
+
+# OpenTelemetry Rust
+
+The Rust [OpenTelemetry](https://opentelemetry.io/) implementation.
 
 [![Crates.io: opentelemetry](https://img.shields.io/crates/v/opentelemetry.svg)](https://crates.io/crates/opentelemetry)
 [![Documentation](https://docs.rs/opentelemetry/badge.svg)](https://docs.rs/opentelemetry)
 [![Crates.io](https://img.shields.io/crates/l/opentelemetry)](./LICENSE)
```

Comment:
This didn't change in this PR, but I noticed it in the other readmes. Can this alt text be something like "License"?

```suggestion
[![License](https://img.shields.io/crates/l/opentelemetry)](./LICENSE)
```